Client/src/Assets/emptyJSFile.json

```diff
@@ -0,0 +1,96 @@
+{
+    "reference": {
+        "type": "",
+        "publisher": "",
+        "authors": [
+            {
+                "firstname": "",
+                "middlename": "",
+                "lastname": ""
+            },
+            {
+                "firstname": "",
+                "middlename": "",
+                "lastname": ""
+            }
+        ],
+        "title": "",
```

Comment:
After title there should be a "doi": "" field

Reply:
addressed on the latest commit. thanks for the feedback